repose-aggregator/components/filters/scripting-filter/pom.xml

```diff
@@ -0,0 +1,142 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.openrepose</groupId>
+        <artifactId>filters-support</artifactId>
+        <version>8.0.0.0-SNAPSHOT</version>
+    </parent>
+
+    <name>Repose Components - Scripting</name>
+    <artifactId>scripting-filter</artifactId>
+    <packaging>jar</packaging>
+
+    <description>
+        This filter will run user-provided scripts in supported scripting languages.
+        These scripts will have access to the request and response objects.
+    </description>
+
+    <properties>
+        <sonar.jacoco.itReportPath>${project.basedir}/../../target/jacoco-it.exec</sonar.jacoco.itReportPath>
+    </properties>
+
+    <dependencies>
+        <!-- Internal dependencies -->
+        <dependency>
+            <groupId>org.openrepose</groupId>
+            <artifactId>core-service-api</artifactId>
```

Comment:
Should be repose-core-api.  I bet this is the reason for the Maven build failure and the extra functional test failures.